### PR TITLE
Optional larger icons & icons in info popover

### PR DESCRIPTION
**Pull request by parkershepherd**
This pull request includes a

- [ ] Bug fix
- [x] New feature
- [ ] Translation

The following changes were made

- Added large icons
- Added toggle to change (pokemon) icon size
- Updated localstorage code so pokemon, pokestops, gyms defaulted to 'true', and large icons defaults to 'false'
- Added icons to pokemon info popover

image

If this is related to an existing ticket, include a link to it as well.

### CODE REVIEW

**Comment by npeep** [comment]
Works beautifully for me.  Looks beautiful too.  So nice to relax my eyes while looking at these little pokemon on my phone screen ;)

**Comment by parkershepherd** [comment]
Just realized 2 other PRs do similar things. Let me know what you'd like me to do with this one

**Comment by prajwalkman** [comment]
@parkershepherd I like this one because of the black outlines :)

**Comment by parkershepherd** [comment]
Updated to use the localstorage PR that was recently merged

**Comment by JakeCooper** [comment]
Jumping from the other PR #1091 . I can't find what you mean by "Accounts for this".

**Comment by parkershepherd** [comment]
@JakeCooper see `map.js` from this commit: https://github.com/AHAAAAAAA/PokemonGo-Map/pull/1052/commits/3a2b7d6637cec24f375567ac98ca972860d5c980

`localStorage.showPokestops !== 'false'` evaluates to `true` when the value is `true` OR `null`, which would make it essentially "default" to `true`

**Comment by JakeCooper** [comment]
Ahh alright, I can dig that :D.

**Comment by hhu94** [comment]
Tested and works, thank you so much for this!

**Comment by malosaa** [comment]
howcan i get this to work with the master ?

**Comment by parkershepherd** [comment]
Anything I need to do to get this merged, @AHAAAAAAA ?

**Comment by jangrewe** [comment]
Oh please dear god (or @AHAAAAAAA), get this merged ASAP - my eyes are hurting because of those tiny icons! :grin:

**Comment by malosaa** [comment]
Yea merge it :D:D:D :+1:

**Comment by JZ6** [comment]
+1 about to merge

**Comment by andrewda** [comment]
This causes an issue where the side nav/filter bar doesn't load (#1310)

File: static/js/app.js [hunk 1]
```diff
@@ -34,8 +34,6 @@
 		$navToggle = document.querySelector('a[href="#nav"]'),
 		$navClose;
 
-	var $overlay = document.querySelector('#trigger-overlay');
-
 	// Event: Prevent clicks/taps inside the nav from bubbling.
```
Comment (JZ6) [inline static/js/app.js]: This is causing the unusable nav button issue, fix please

**Comment by JZ6** [comment]
please fix and do a new pr

**Comment by parkershepherd** [comment]
@JZ6 https://github.com/AHAAAAAAA/PokemonGo-Map/pull/1317 created

**Pull request merged**